packages/cpsuLambda/src/schema/format_1/transformer.ts

```diff
@@ -5,20 +5,30 @@ import {
   requestType,
   deliveryType
 } from "./request"
-import {Ok, Result, collectResult} from "pratica"
+import {
+  collectMaybe,
+  Just,
+  Maybe,
+  Nothing,
+  Ok
+} from "pratica"
 import {v4 as uuidv4} from "uuid"
 import {Transformer} from "../../handler"
-import {wrap_with_status} from "../../utils"
+import "../../utils"
```

Comment:
note: this is so we can use the String.window functionality in the UUID generator